### ISSUE TITLE
Error when using the GET method

### ISSUE THREAD

**Issue by greg-violette** [VolkovLabs/volkovlabs-form-panel#84]
I have a very simple GET method that should return data...confirmed using postman but I get the error below when I try to call it from the panel
'FormPanel.tsx:169 Uncaught (in promise) TypeError: Failed to execute 'set' on 'Headers': Invalid name
    at eval (FormPanel.tsx:169:15)
    at Array.forEach (<anonymous>)
    at eval (FormPanel.tsx:168:29)
    at s (FormPanel.tsx:2:1)
    at Generator.eval [as _invoke] (FormPanel.tsx:2:1)
    at Generator.eval [as next] (FormPanel.tsx:2:1)
    at Mo (FormPanel.tsx:2:1)
    at a (FormPanel.tsx:2:1)
    at eval (FormPanel.tsx:2:1)
    at new Promise (<anonymous>)'

**Comment by mikhail-vl** [comment]
@greg-violette Hello, do you have headers for the Initial Request with empty name in Panel's options? Or is it replaced with Empty Variables?

  Looks like we are missing check for the `parameter.name` and Fetch API gave you `Invalid name` error.
 
```
    options.initial.header?.forEach((parameter) => {
      headers.set(replaceVariables(parameter.name), replaceVariables(parameter.value));
    });

```

**Comment by greg-violette** [comment]
I just have on header and it is a static value.  Here is an excerpt from my json export
`  "contentType": "application/json",
          "header": [
            {
              "name": "X-VSF-API-Token",
              "value": "YTc4ZmU1NGUtYjFkZC00YjlkLWI3M2YtZWNkZmFiZDE3MWQ0"
            }
          ],
          "highlight": false,`

**Comment by mikhail-vl** [comment]
@greg-violette Thank you. I will take a look today and get back to you shortly.

**Comment by greg-violette** [comment]
OK, thanks.  On another note, is there any dcoumentation on the Code Editor data type?  I need to pass in  a json array of objects and I was thinking the Code Editor may be the way to do that but I'm getting errors calling my API endpoint.
Thanks.

**Comment by mikhail-vl** [comment]
@greg-violette I reproduced the error when added Header Parameters has empty `name` field. Fixed in #85.

    Code Editor is implemented using Monaco Code Editor, which is a part of the Grafana. You can choose Language `JSON` or `Javascript`. I just tested using JSON and it works as expected converting text into string like:
```
{
    "code": "<html>\n  <div>\n  </div>\n</html>\n"
}
```

  Please let me know if there is anything else.

**Comment by mikhail-vl** [comment]
@greg-violette Fix for Request Header merged and will be a part of the upcoming release.

Please let me know if you able to fix the Code Editor errors and if there is anything else.
